tests/system/runtimes/test_notifications.py

```diff
@@ -75,6 +75,96 @@ def _assert_notifications():
             _assert_notifications,
         )
 
+    def test_set_run_notifications(self):
+
+        notification_name = "slack-should-succeed"
+
+        def _assert_notification_was_sent():
+            runs = self._run_db.list_runs(
+                project=self.project_name,
+                with_notifications=True,
+            )
+            assert len(runs) == 1
+            assert len(runs[0]["status"]["notifications"]) == 1
+            assert (
+                runs[0]["status"]["notifications"][notification_name]["status"]
+                == "sent"
+            )
+
+        self._create_sleep_func_in_project()
+
+        notification = self._create_notification(
+            name=notification_name,
+            message="should-succeed",
+            params={
+                # dummy slack test url should return 200
+                "webhook": "https://slack.com/api/api.test",
+            },
+        )
+
+        run = self.project.run_function(
+            "test-sleep", local=False, params={"time_to_sleep": 10}
+        )
+        self._run_db.set_run_notifications(
+            self.project_name, run.metadata.uid, [notification]
+        )
+
+        run.wait_for_completion()
+
+        # the notifications are sent asynchronously, so we need to wait for them
+        mlrun.utils.retry_until_successful(
+            1,
+            40,
+            self._logger,
+            True,
+            _assert_notification_was_sent,
+        )
+
+    # TODO: currently the test checks setting notifications on the schedule but it doesn't check that the notification
+    #  has been sent. We need to configure the system test mlrun system config to allow schedules to run more frequently
+    #  than once per 10 minutes, and then we can check that the notification has been sent.
```

Comment:
We used to override configurations for system test setups such that minimal schedule interval would be 1 minute, IIRC. Is this not does anymore?

Reply:
Not currently sadly. Checked on the system